Packs/QuestKace/Integrations/QuestKace/QuestKace.py

```diff
@@ -0,0 +1,785 @@
+            return self._http_request("GET", url_suffix=f"/inventory/machines?filtering={filter_fields}",
+                                      headers=headers)
+        else:
+            return self._http_request("GET", url_suffix=f"/inventory/machines", headers=headers)
+
+    def assets_list_request(self, filter_fields: Optional[str] = None) -> dict:
+        """List of assets.
+           Returns:
+               Response from API.
+        """
+        headers = {
+            'Accept': 'application/json',
+            'x-dell-csrf-token': self._token,
+            'x-dell-api-version': '5',
+            'Cookie': self._cookie
+        }
+        if filter_fields:
+            return self._http_request("GET", url_suffix=f"/asset/assets?filtering={filter_fields}", headers=headers)
+        else:
+            return self._http_request("GET", url_suffix="/asset/assets", headers=headers)
+
+    def queues_list_request(self, filter_fields: Optional[str] = None) -> dict:
+        """List of queues.
+           Returns:
+               Response from API.
+        """
+        headers = {
+            'Accept': 'application/json',
+            'x-dell-csrf-token': self._token,
+            'x-dell-api-version': '5',
+            'Cookie': self._cookie
+        }
+        if filter_fields:
+            return self._http_request("GET", url_suffix=f"/service_desk/queues?shaping=fields all"
+                                                        f"&filtering={filter_fields}", headers=headers)
+        else:
+            return self._http_request("GET", url_suffix="/service_desk/queues?shaping=fields all", headers=headers)
+
+    def queues_list_fields_request(self, queue_number: str) -> dict:
+        """List of fields in specific queue.
+            Args:
+                queue_number: queue nubmer for the request.
+           Returns:
+               Response from API.
+        """
+        headers = {
+            'Accept': 'application/json',
+            'x-dell-csrf-token': self._token,
+            'x-dell-api-version': '5',
+            'Cookie': self._cookie
+        }
+        return self._http_request("GET", url_suffix=f"/service_desk/queues/{queue_number}/fields", headers=headers)
+
+    def tickets_list_request(self, shaping_fields: str = None, filter_fields: str = None) -> dict:
+        """List of Tickets.
+            Args:
+                shaping_fields: str of the shaping that will be sent in the request.
+                filter_fields: str of filter that will be sent in the request.
+           Returns:
+               Response from API.
+        """
+        url = "/service_desk/tickets"
+        if not shaping_fields:
+            shaping_fields = set_shaping(self)
+        self.update_token()
+        headers = {
+            'Accept': 'application/json',
+            'x-dell-csrf-token': self._token,
+            'x-dell-api-version': '5',
+            'Cookie': self._cookie
+        }
+        if filter_fields:
```

Comment:
let's build the `url_suffix` accordingly to the `filter_fields`
and use only 1 return in the function